





<div>

  <p><font size="4" face="Arial"><b>Complex networks in biology: enzymes 
- metabolic pathways network:</b></font> <br></p>
<p><font size="3" face="Arial"><b>Description: </b>
Enzymes are proteins that catalyze (<i>i.e.</i> accelerate) chemical 
reactions. In these reactions, the molecules at the beginning of the 
process are called substrates, and the enzyme converts them into different 
molecules, the products. Almost all processes in the cell need enzymes 
in order to occur at significant rates. A metabolic pathway is a series 
of chemical reactions occurring within a cell, catalyzed by enzymes, 
resulting in either the formation of a metabolic product to be used 
or stored by the cell, or the initiation of another metabolic pathway. 
Since enzymes are extremely selective for their substrates and speed 
up only a few reactions from among many possibilities, the set of enzymes 
made in a cell determines which metabolic pathways occur in that cell.</font> <br>

 <br></p>
<p><font size="3" face="Arial"><b>Abstract</b>: This network consists 
of all enzymes who take part in 70 metabolic pathways will be the nodes 
of the network. If two particular enzymes take part in same metabolic 
pathway then there will be link between them. The weight of the link 
is equal to the number of metabolic pathways they both participate. 
Different organisms will pose different networks. We will take some 
of these networks, analyze those individually and correlate them. We 
will try to find some evidence of evolution among the considered organisms.</font> <br>
</p>
<p><font size="3" face="Arial">There are other ways to represent the 
same data. We will construct bipartite networks from same data where 
bottom nodes will be the enzymes and top nodes will be the metabolic 
pathways. Assuming the bottom set of nodes is constant, we will analyze 
the different properties (i.e. degree distribution) of bottom set of 
nodes and their changing pattern among different organisms. And will 
try to find some evidence of evolution. </font> <br></p>
<p><font size="3" face="Arial">Reference – </font></p>
<p><font size="3" face="Arial">Jeong H, Tombor B, Albert R, Oltvai ZN, 
Barabasi AL. The large-scale organization of metabolic networks.</font><font size="2" face="Times New Roman"> </font><font size="3" face="Arial">Nature. 
2000 Oct 5;407(6804):651-4.</font><font size="2" face="Times New Roman"> </font><font size="3" face="Arial">PMID: 11034217 [PubMed - indexed for MEDLINE]</font></p>

<p><a href="http://www.nd.edu/~networks/Publication+Categories/03+Journal+Articles/Biology/Large-Scale+MetabolicNetworks_Nature+407,+651-654+(2000).pdf" target="_blank"><font size="3" face="Arial"><u>http://www.nd.edu/~networks<WBR>/Publication%20Categories/03<WBR>%20Journal%20Articles/Biology<WBR>/Large-Scale%20MetabolicNetwork<WBR>s_Nature%20407,%20651-654%20<WBR>(2000).pdf</u></font></a> <br>
</p>
<p><font size="3" face="Arial">Light S, Kraulis P, Elofsson A. Preferential 
attachment in the evolution of metabolic networks.</font><font size="2" face="Times New Roman"> </font><font size="3" face="Arial">BMC Genomics. 2005 Nov 10;6:159. PMID: 16281983 
[PubMed - indexed for MEDLINE]</font></p>
<p><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16281983" target="_blank"><font size="3" face="Arial"><u>http://www.pubmedcentral.nih<WBR>.gov/articlerender.fcgi?tool<WBR>=pubmed&amp;pubmedid=16281983</u></font></a> <br>

 <br></p>
<p><font size="3" face="Arial">Data - </font><a href="http://www.genome.jp/kegg/pathway.html#metabolism" target="_blank"><font size="3" face="Arial"><u>http://www.genome.jp/kegg<WBR>/pathway.html#metabolism</u></font></a></p>
<p><font size="3" face="Arial">(1.1 to 1.6 = 70 metabolic pathways, 
enzymes are linked to a subset of the genes and enumerated by the EC 
code shown in the boxes after clicking on a pathway name, after <br>
organism selection from the menu above a pathway diagram the pathways 
show in green color those EC numbers that are linked to the selected 
pathway in the selected organism, EC numbers are uniform across all 
organisms, no ortholog table required)</font> <br></p>


</div>

</div></body></html>
